### PR TITLE
Emotion 11 Upgrade (v2)

### LINKED ISSUE #8348: Upgrade @emotion/core and @emotion/styled to new major version 11 in Simorgh
**Is your feature request related to a problem? Please describe.**
Upgrade `@emotion/core` and `@emotion/styled` to new major version 11.

Psammead PR 1: https://github.com/bbc/psammead/pull/4193
Psammead PR 2 (psammead-styles): https://github.com/bbc/psammead/pull/4231

Details of any breaking changes can be found here https://github.com/emotion-js/emotion/releases/tag/%40emotion%2Fstyled%4011.0.0

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves https://github.com/bbc/simorgh/issues/8348

**Overall change:**
_Upgrade Simorgh to Emotion 11._

**Code changes:**

- _Update all psammead and gel packages to latest version._
- _Update Simorgh to Emotion 11._
- _Rewrite 3 test suites that are not compatible with Emotion 11 due to them mocking React._
- _Update webpack config to alias 3 packages that were being duplicated - this is a temporary holdover until we migrate to Yarn._
- _Update storybook webpack config to deal with storybook 6 not supporting emotion 11._

Snapshot Updates: https://github.com/bbc/simorgh/pull/8852
Prior PR: https://github.com/bbc/simorgh/pull/8742

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by jroebu14 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -68,58 +68,58 @@
...
-    "@bbc/psammead-visually-hidden-text": "2.0.1",
+    "@bbc/psammead-amp-geo": "1.2.2",
+    "@bbc/psammead-assets": "3.1.5",
+    "@bbc/psammead-brand": "7.0.30",
+    "@bbc/psammead-bulleted-list": "3.0.12",
+    "@bbc/psammead-bulletin": "5.0.30",
+    "@bbc/psammead-byline": "3.0.13",
+    "@bbc/psammead-calendars": "2.0.23",
+    "@bbc/psammead-caption": "4.1.8",
+    "@bbc/psammead-consent-banner": "5.3.2",
```
Comment (ryanmccombe) [inline package.json:81]: Major bump here - checking w/ engage - https://github.com/bbc/psammead/blob/latest/packages/components/psammead-consent-banner/CHANGELOG.md

Comment (ryanmccombe) [reply package.json:81]: Can't update this 😦 - this PR is blocked for the minute - can't update `psammead-consent-banner` to `5.x` as it has a11y issues that Engage are working on; can't update to emotion 11 with this component on `4.x` as the emotion 10 + emotion 11 combination causes style tags in the body of the document, thereby failing AMP validation

https://github.com/bbc/simorgh/pull/8827

File: webpack.config.js [hunk 2]
```diff
@@ -48,6 +48,18 @@ module.exports = (shell = {}) => {
...
+          'node_modules/@emotion/serialize',
+        ),
+        '@emotion/utils': path.resolve(
+          __dirname,
+          'node_modules/@emotion/utils',
+        ),
+        '@emotion/sheet': path.resolve(
+          __dirname,
+          'node_modules/@emotion/sheet',
+        ),
```
Comment (andrewscfc) [inline webpack.config.js:62]: Why are these aliases necessary?

**Review by andrewscfc — APPROVED** [review]
Just compared against the release notes and all looks good: https://emotion.sh/docs/emotion-11

Thanks for sticking with this

**Comment by andrewscfc** [comment]
Still a few snapshots to update I think

File: webpack.config.js (hunk 2 above)
Comment (ryanmccombe) [reply webpack.config.js:62]: they're to deal with some duplicate dependencies - will be removed again once the yarn PRs are merged

**Pull request merged**